infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -236,7 +236,7 @@ public TokenType txnCanCommit(TxResolutionInfo txInfo, /** Input. */
                 // for each key pair, check for conflict;
                 // if not present, check against the wildcard
                 conflictParamSet.forEach(conflictParam -> {
-                    int conflictKeyHash = getConflictHashCode(entry.getKey(),
+                    String conflictKeyHash = getConflictHashCode(entry.getKey(),
```

Comment:
Looks fine but it is not solving Anny's case. Do you plan to introduce the bigger hash in the next PR.
Also do you need more tests as the hashcode is now a string ?